contracts/src/everlong/EverlongPositions.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IEverlongPositions } from "../interfaces/IEverlongPositions.sol";
+import { DoubleEndedQueue } from "openzeppelin/utils/structs/DoubleEndedQueue.sol";
+
+contract EverlongPositions is IEverlongPositions {
```

Comment:
Let's add contract-level Natspec to all of these contracts. This would explain what the contract is used for. In this case, it should clearly identify what positions are stored in Everlong and what they are used for.